File: src/main/java/org/mtransit/android/data/StatusProviderProperties.java
package org.mtransit.android.data;

public class StatusProviderProperties {

	private String targetAuthority;
	private String authority;

	public StatusProviderProperties(String authority, String targetAuthority) {
		this.authority = authority;
		this.targetAuthority = targetAuthority;
	}

	public String getTargetAuthority() {
		return this.targetAuthority;
	}
	
	public String getAuthority() {
		return authority;
	}
	
	@Override
	public String toString() {
		return new StringBuilder().append(StatusProviderProperties.class.getSimpleName()).append('{') //
				.append("authority:").append(authority).append(',') //
				.append("targetAuthority:").append(targetAuthority).append(',') //
				.append('}').toString();
	}

}

File: src/main/java/org/mtransit/android/data/StatusProviderProperties.kt
package org.mtransit.android.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "status_provider_properties")
data class StatusProviderProperties(
    @PrimaryKey
    @ColumnInfo(name = "authority")
    val authority: String,
    @ColumnInfo(name = "target_authority")
    val targetAuthority: String
)